shell/list/fleet.cattle.io.clusterregistrationtoken.vue

```diff
@@ -25,8 +25,11 @@ export default {
   },
 
   async fetch() {
+    await this.$fetchType(FLEET.TOKEN);
```

Comment:
Do we need this? Is the resource not fleet token? i.e. will be fetched by the line below?